### PR TITLE
Resolved Bug 661 : Misplaced Position of Setting Icon on Sidebar

### LINKED ISSUE #661: Local environment > Design system > side nav bar > setting icon> misplaced the position of setting icon

### PR DESCRIPTION
**Pull request by poojagurav21**
## Screenshots :
image

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .storybook/custom-theme.css [hunk 1]
```diff
@@ -82,4 +82,10 @@ nav.container.sidebar-container.css-t68aeq {
     top: -107px !important;
     left: -10px !important;
     position: relative !important;
+}
+
+.css-1rb1jn6 {
```
Comment (Ketan-Khalate) [inline .storybook/custom-theme.css:87]: @poojagurav21 why this class is added please confirm on this

Comment (poojagurav21) [reply .storybook/custom-theme.css:87]: Reduced the size of Raaghu logo. Confirm with Kaushik.

**Comment by poojagurav21** [comment]
@Ketan-Khalate  Created New pr for this work item as it has some file commits issue. 
Please refer Pr: https://github.com/Wai-Technologies/raaghu-react/pull/722

File: .storybook/custom-theme.css (hunk 1 above)
Comment (hemalatagaikwad) [reply .storybook/custom-theme.css:87]: @poojagurav21 , please resolved conflicts

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
Please resolved conflicts

**Comment by poojagurav21** [comment]
@hemalatagaikwad This pr needs to be closed as I have created this branch from stage.

**Comment by hemalatagaikwad** [comment]
already create another pr , https://github.com/Wai-Technologies/raaghu-react/pull/722
so close this pr

**Pull request closed without merging**